src-web/components/kappnav/modals/NavModal.js

```diff
@@ -444,9 +466,25 @@ class NavModalForm extends React.PureComponent {
   getMenuItems() {
     const { menuItems, onMenuClick, validationErrors } = this.props
     return menuItems.map((item, index) =>
-      <li key={index} className={this.isActive(item)} id={item}>
-        <a href="#" onClick={onMenuClick.bind(null, item)} className='menu-item' role='menuitem'>{msgs.get(`modal.nav.${item}`)}</a>
-        {validationErrors.form && validationErrors[item] && !this.isActive(item) && <Icon className='modal-tab-error' name='icon--error--glyph' description={msgs.get('svg.description.error')} />}
+      <li key={index} className={this.isActive(item.label)} id={item.label}>
+        <a href="#" onClick={onMenuClick.bind(null, item.label)} className='menu-item' role='menuitem'>{msgs.get(`modal.nav.${item.label}`)}</a>
+        {validationErrors.form && validationErrors[item.label] && !this.isActive(item.label) && <Icon className='modal-tab-error' name='icon--error--glyph' description={msgs.get('svg.description.error')} />}
+        {/* adding tooltip */}
+        <div className ="tooltip-margin">
+          <Tooltip
+            iconDescription={msgs.get('formtip.tooltip')}
+            triggerClassName="tooltip-active-hover"
+            triggerText=''
+            direction='right'
+            showIcon={true}>
+            <p dangerouslySetInnerHTML={{ __html: item.tooltip }} />
+            <div className="bx--tooltip__footer">
+              <a className="bx--link" target="_blank" href={this.props.learnmoreURL}>
```

Comment:
We need to add rel=”noopener noreferrer” to the <a> tag because without it we have a security vulnerability.